src/wallet/api/wallet.cpp

```diff
@@ -1608,7 +1608,9 @@ bool WalletImpl::setCacheAttribute(const std::string &key, const std::string &va
 
 std::string WalletImpl::getCacheAttribute(const std::string &key) const
 {
-    return m_wallet->get_attribute(key);
+    std::string value;
+    m_wallet->get_attribute(key, value);
+    return value;
```

Comment:
Please add /monero#5779 to the end of the commit message.

Reply:
Resolved.